test/suite/test_hs02.py

```diff
@@ -40,6 +41,14 @@ class test_hs02(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=50MB,log=(enabled)'
     session_config = 'isolation=snapshot'
 
+    key_format_values = [
+        ('string', dict(key_format='S')),
+        # Remove for column store until rollback to stable is implemented for column store
```

Comment:
Generally, we preface these types of comments with a `FIXME-WT-XXXX`. Perhaps consider rewording as follows:

`# FIXME-WT-XXXX: Uncomment the column store scenario when rollback to stable support is complete.`

Also, WT-5545 is not the correct ticket to reference here. I believe it should be WT-7120.

Reply:
No worries, just changed all our other comments to match this format.